src/sass/_variables.scss

```diff
@@ -9,3 +9,10 @@ $primary: #d58e32;
 $text-color: #2a2a2a;
 
 $form-border-color: #292929;
+
+$animation-time-bg: 0.25s;
+$animation-time-font: 0.25s;
+
+%elementsAnimation {
```

Comment:
to be consistent you could move this to a separate file named `_placeholders.scss` or sth